### DISCUSSION TITLE
Discord community

### DISCUSSION THREAD

**Discussion by gucio321** [AllenDang/giu#910] [Ideas]
@AllenDang can I make an experiment - create a discord server put it in readme and check if it will gain any members :smile:? 
I think that a server for giu and cimgui-go should be a nice alternative to gh issues for beginner-questions e.t.c.
(I can handle this so you don't need to join if you don't want to)

**Comment by AllenDang** [comment]
@gucio321 Sure thing! Please go ahead :)

**Reply by gucio321** [reply]
ok, here we go! https://discord.gg/Tt7eq6YKQS
